pimd/pim_cmd.c

```diff
@@ -7859,7 +7859,7 @@ DEFUN (interface_no_ip_igmp,
        IFACE_IGMP_STR)
 {
 	const struct lyd_node *pim_enable_dnode;
-	char pim_if_xpath[XPATH_MAXLEN + 20];
+	char pim_if_xpath[XPATH_MAXLEN + 128];
```

Comment:
where is this number coming from? Adding in a magic number here defeats the purpose of having the macro

Reply:
It's coming from the length of the fixed text. `XPATH_MAXLEN` is how long `VTY_CURR_XPATH` can be, and we're adding some fixed text to that, which is less than 128 characters long.

Reply:
what are we adding 128 @eqvinox  ?

Reply:
to give space for the pim attribute itself ?

Reply:
if it's a fixed value seems like that should be derived from somewhere rather than just pulling it out air to avoid having to change this code again in the future if the data changes, but I will lift my nack nonetheless